.github/workflows/integration_tests.yml

```diff
@@ -86,6 +86,9 @@ jobs:
     steps:
     - name: Checkout code
       uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+    - uses: actions/setup-go@331ce1d993939866bb63c32c6cbbfd48fa76fc57
+      with:
+        go-version: '1.16'
```

Comment:
We're already using go 1.16 in [go.mod](https://github.com/linkerd/linkerd2/blob/a866aa416c09fafdf384eaba3ac8aa5c74678c19/go.mod#L3), but the ubuntu 20.04 image defaults to go 1.15. This ensures we use go 1.16 which is required for upgrading these dependencies.